apps/course/src/app/types/legal.type.ts

```diff
@@ -0,0 +1,11 @@
+export type Legal = {
+  name: string;
+  description: string;
+  file: string;
```

Comment:
Make a nested type `file: LegalFile[]`

Reply:
I don't understand please explain a little more in detail

Reply:
Your `legal.type.ts` file should look like this to meet best practices and correct typing:
export type Legal = {
  name: string;
  description: string;
  file: LegalFile[];
}
  
export type LegalFile = {
   name: string;
   description: string;
   file: string;
}

Reply:
By doing that I get a lot of issues in my Service